package.json

```diff
@@ -14,16 +14,16 @@
     "test:lint": "standard",
     "test:unit": "tape -r './src/test/unit/setup' 'src/@(web|config)/**/*.test.js'",
     "test:unit:coverage": "nyc npm run test:unit",
-    "test:acceptance": "mkdir -p build/reports && cross-env TESTS=acceptance cucumber-js src/test/acceptance/features/ --tags 'not @ignore' --require 'src/test/common/steps/*.js' --require 'src/test/acceptance/config' -f json:build/reports/acceptance-report.json",
-    "test:acceptance-no-wiremock": "mkdir -p build/reports && cross-env TESTS=acceptance cucumber-js src/test/acceptance/features/ --tags 'not @ignore' --tags 'not @RequiresWiremock' --require 'src/test/common/steps/*.js' --require 'src/test/acceptance/config' -f json:build/reports/acceptance-report.json",
-    "test:acceptance:debug": "mkdir -p build/reports && cross-env TESTS=acceptance node $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/acceptance/features/ --tags 'not @ignore' --require 'src/test/common/steps/*.js' --require 'src/test/acceptance/config' -f json:build/reports/acceptance-report.json",
+    "test:acceptance": "mkdir -p build/reports && cross-env TESTS=acceptance cucumber-js src/test/acceptance/features/navigation.feature --tags 'not @ignore' --require 'src/test/common/steps/*.js' --require 'src/test/acceptance/config' --require 'src/test/common/timeout.js' -f json:build/reports/acceptance-report.json",
```

Comment:
should this have `navigation.feature` in the command?

Reply:
Removed.